### DISCUSSION TITLE
Any chance add more Korean channels?

### DISCUSSION THREAD

**Discussion by MarkJP0** [iptv-org/community#1012] [Q&A]
Will there any chance Korean channel will be add more? Like JTBC, Mnet etc... 

Also will some links from the Korean playlist be fix like some of them arent working anymore like KBS 1HD or the SPOTV

Thanks in advance.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
